examples/consumergroup/main.go

```diff
@@ -155,10 +156,17 @@ func (consumer *Consumer) ConsumeClaim(session sarama.ConsumerGroupSession, clai
 	// Do not move the code below to a goroutine.
 	// The `ConsumeClaim` itself is called within a goroutine, see:
 	// https://github.com/Shopify/sarama/blob/master/consumer_group.go#L27-L29
+	id := goroutineID()
 	for message := range claim.Messages() {
-		log.Printf("Message claimed: value = %s, timestamp = %v, topic = %s", string(message.Value), message.Timestamp, message.Topic)
+		log.Printf("Message claimed: value = %s, goroutineID = %s, timestamp = %v, topic = %s, partition = %d", string(message.Value), id, message.Timestamp, message.Topic, message.Partition)
 		session.MarkMessage(message, "")
 	}
 
 	return nil
 }
+
+func goroutineID() string {
+	var buf [64]byte
+	n := runtime.Stack(buf[:], false)
+	return strings.Fields(string(buf[:n]))[1]
+}
```

Comment:
I haven't used this before? is this a recommended practice?

it seems go folks doesn't like this:
https://github.com/golang/go/issues/27361#issuecomment-422899200

Reply:
You are right. It's not a recommended practice to try to get the goroutine id.

This PR may be unnecessary because the comment above has stated that the ConsumeClaim() function would be called in a separate goroutine.